conventions/README.md

```diff
@@ -161,136 +204,152 @@ Frameworks can use their native tools to interact with the provided `<input>` wh
 
-**Implemented In:**
+## CSS Class Names
 
-* [`<calcite-tab-nav>`](../src/components/calcite-tab-nav/calcite-tab-nav.tsx);
+Because most components utilize shadow DOM, there is far less concern over naming collisions in a global CSS namespace. In addition, it's better for file transfer times and easier to write if class names are shorter. For these reasons, full BEM is not necessary. Instead, we can omit the "Block", and use the host instead. Consider the following BEM markup:
 
-## CSS Class Names
+```html
+<div class="card">
+  <h3 class="card__title card__title--large">Title</h3>
+  <p class="card__text">Text</p>
+</div>
+```
 
-@TODO Discuss BEM
+In a component using shadow DOM, this should instead be written as:
 
-* https://github.com/ArcGIS/calcite-components/issues/28
-* https://github.com/ArcGIS/calcite-components/pull/24#discussion_r287462934
-* https://github.com/ArcGIS/calcite-components/pull/24#issuecomment-495788683
-* https://github.com/ArcGIS/calcite-components/pull/24#issuecomment-497962263
+```jsx
+<Host>
+  <h3 class="title title--large">Title</h3>
+  <p class="text">Text</p>
+</Host>
+```
 
-## a11y
+Notice `.card` does not appear anywhere. We would then apply styles to the host element itself:
 
-In generally follow the guidelines and standards in these articles:
+```scss
+:host {
+  // card styles here
+}
+
+.title {
+}
+.title--large {
+}
+.text {
+}
+```
 
-* [Google Accessibility Overview](https://developers.google.com/web/fundamentals/accessibility/)
-* [WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)
+Modifier classes on the "block" (host element) can often be written by reflecting the prop and selecting it directly via an attribute selector:
 
-**Implemented In:**
+```scss
+:host([color="blue"]) {
+}
+```
 
-* [`<calcite-tabs>`](../src/components/calcite-tabs/calcite-tabs.tsx);
-* [`<calcite-tab-nav>`](../src/components/calcite-tab-nav/calcite-tab-nav.tsx);
-* [`<calcite-tab>`](../src/components/calcite-tab/calcite-tab.tsx);
-* [`<calcite-tab-title>`](../src/components/calcite-tab-title/calcite-tab-title.tsx);
+This builds a nice symmetry between the styling and the public API of a component.
+
+- https://github.com/ArcGIS/calcite-components/issues/28
+- https://github.com/ArcGIS/calcite-components/pull/24#discussion_r287462934
+- https://github.com/ArcGIS/calcite-components/pull/24#issuecomment-495788683
+- https://github.com/ArcGIS/calcite-components/pull/24#issuecomment-497962263
+
+## a11y
+
+In generally follow the guidelines and standards in these articles:
+
+- [Google Accessibility Overview](https://developers.google.com/web/fundamentals/accessibility/)
+- [WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)
 
 ## i18n
 
 Components should require as a few text translations as possible. In general lets users supply text values via slots and attributes. The lets user handle translations with their apps.
```

Comment:
Suggested copy change:
> Components should require as a few text translations as possible. In general lets users supply text values via slots and attributes. The lets user handle translations with their apps.

Components should require as a few text translations as possible. In general, components should let users supply text values via slots and attributes; this lets a user handle translation within their apps.